packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt

```diff
@@ -68,7 +68,7 @@ class ImagePickerModule : Module() {
       launchContract({ imageLibraryLauncher.launch(contractOptions) }, options)
     }
 
-    AsyncFunction("getPendingResultAsync") Coroutine { _: Promise ->
+    AsyncFunction("getPendingResultAsync") Coroutine { ->
```

Comment:
Do we even need the arrow? 😄 
```suggestion
    AsyncFunction("getPendingResultAsync") Coroutine {
```

Reply:
I guess we do need it, because without this `->` kotlin gets dizzy and tries to match every overloaded signature available (starting from no-params through all the parametrised ones). It needs explicit instruction about no params 😉